tests/unit/views/table.vue.spec.js

```diff
@@ -17,11 +17,18 @@
 
 import { mount } from '@vue/test-utils'
 import { createStore } from 'vuex'
+import { createVuetify } from 'vuetify'
 import sinon from 'sinon'
 import storeOptions from '@/store/options'
 import Table from '@/views/Table.vue'
 import WorkflowService from '@/services/workflow.service'
 import User from '@/model/User.model'
+import { nextTick } from 'vue'
+import CylcObjectPlugin from '@/components/cylc/cylcObject/plugin'
+import { simpleTableTasks } from '@/../tests/unit/components/cylc/table/table.data'
```

Comment:
These are being loaded from `@/../tests/unit/components/` which is maybe a bit weird as the test is in 'tests/unit/views' should simple tasks be stored somewhere more centrally?

Reply:
So long as it's clear what code does and doesn't belong to the tests, I don't mind.